modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/edit_folder.jsp

```diff
@@ -486,34 +490,42 @@ renderResponse.setTitle(headerTitle);
 
 			searchContainer.deleteRow(tr, link.getAttribute('data-rowId'));
 
-			A.one(
-				'#<portlet:namespace />defaultFileEntryTypeId-' +
+			var option = document.getElementById(
+				'<portlet:namespace />defaultFileEntryTypeId-' +
 					link.getAttribute('data-rowId')
-			).remove();
+			);
+
+			option.parentElement.removeChild(option);
 
 			<portlet:namespace />documentTypesChanged = true;
 
-			var select = A.one(
-				'#<%= liferayPortletResponse.getNamespace() + "workflowDefinition" + DLFileEntryTypeConstants.FILE_ENTRY_TYPE_ID_ALL %>'
+			var select = document.getElementById(
+				'<%= liferayPortletResponse.getNamespace() + "workflowDefinition" + DLFileEntryTypeConstants.FILE_ENTRY_TYPE_ID_ALL %>'
 			);
 
-			var selectContainer = A.one(
-				'#<portlet:namespace />restrictionTypeWorkflow'
+			var selectContainer = document.getElementById(
+				'<portlet:namespace />restrictionTypeWorkflow'
 			);
 
-			var fileEntryTypesCount = select.get('children').size();
+			var fileEntryTypesCount = select.children.length;
 
 			if (fileEntryTypesCount == 0) {
-				selectContainer.hide();
+				selectContainer.classList.add('hide');
```

Comment:
(No need to update this pull now, but for the future) there's an `==` that could be switched to `===` here, so keep your eyes open for things like that as you drive through older files. As you know, some of these will be autofixed when we roll out https://github.com/liferay/eslint-config-liferay/pull/171 (and this would be one of them), but here are others that will require manual review, so it's a good idea to fix stuff as we see it.